test/setup.cfg

```diff
@@ -3,6 +3,7 @@ minversion = 3.6
 markers =
     bashcomp
     complete
+xfail_strict=true
```

Comment:
If we want this, I think we also want/need to go all things that can be xfails or xpasses, and mark them as such, _specifically on per distro basis_ (i.e. semantically for example like "expected to fail on CentOS 7").

We could also consider changing some xfails to skips, but to me the downside of that would be that we'd get no indication if something that used to fail has started to pass due to something (which xpass gives us). There might be some skips that we'd want to convert to xfails for this reason.